app/models/session.php

```diff
@@ -0,0 +1,12 @@
+<?php
+class Session extends AppModel
+{
+    public static function setSession($session_name, $value)
+    {
+        $_SESSION["$session_name"] = $value;
+    }
+    public static function getSession($session_name)
+    {
+        return $_SESSION["$session_name"];
+    }
+}
```

Comment:
I tried using set() only but it appears to have an error, so I change it to setSession() then it works. I think it is a reserved keyword? :bow: